src/tess_ephem/ephem.py

```diff
@@ -50,9 +50,9 @@ def __init__(
             location=location,
         )
         self._raf = create_angle_interpolator(
-            eph["datetime_jd"], eph["RA_app"], enforce_positive=True
+            eph["datetime_jd"], eph["RA"], enforce_positive=True
         )
-        self._decf = create_angle_interpolator(eph["datetime_jd"], eph["DEC_app"])
+        self._decf = create_angle_interpolator(eph["datetime_jd"], eph["DEC"])
```

Comment:
Can you document this change in the PR from apparent to astrometric and what difference it makes?

Reply:
These two coordinates are explained [here](https://ssd.jpl.nasa.gov/horizons/manual.html). I did some tests with five bright asteroids (V<18) in sector 6, camera 1, ccd 1. The apparent and astrometric RA, Dec resulted in an offset of about one pixel in the column position of each asteroid. The astrometric position showed much better agreement with the flux centroid than the apparent position. Christina suggested that this might be because the corrections used to calculate the apparent position have already been done by the SPOC pipeline when creating the WCS, hence using the apparent position is essentially correcting twice. I would like to try and find a definitive answer about which coordinates are the correct ones to use in this case. Would you like me to write a comment about this somewhere?

Reply:
I understand, this explanation here is fine. If you find something more definitive you can add it here for later reference.

Reply:
<img width="1393" alt="astrometric_apparent_coords" src="https://github.com/user-attachments/assets/9a069ab6-8a73-4e56-8c7d-182b622c18ae">
This figure shows four example bright asteroids from TESS sector 6, camera 1, ccd 1. The astrometric (upper) and apparent (lower) coordinates are indicated as a pink dot on the FFI image, where you can see the astrometric agrees better with the flux centroid.